policyengine_canada/variables/gov/provinces/sk/tax/income/credits/slic/slic_base.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class slic_base(Variable):
```

Comment:
all province/territory variables start with two-letter code
```suggestion
class sk_slic_base(Variable):
```